engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -184,4 +185,20 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
         }
       }
     }
-}
\ No newline at end of file
+
+  def backendRoute =
+    path("workflows" / Segment / "backends") { version =>
+      get {
+        complete {
+          backendResponse
```

Comment:
I'd at least put a comment here if you think it should stay like this.  As you've observed, things tend to get copy/pasted; it would be unfortunate if this was taken as the right way to write these endpoints generally.